api/applications/views.py

```diff
@@ -98,3 +105,31 @@ def perform_update(self, serializer):
         serializer.validated_data['owner'] = self.request.user
         # TODO: Write code to transfer ownership
         serializer.save(owner=self.request.user)
+
+
+class ApplicationReset(generics.RetrieveUpdateDestroyAPIView, ApplicationMixin):
+    """
+    Get information about a specific API application (eg OAuth2) that the user has registered
```

Comment:
This docstring is a straight copy-paste. Please make it relevant to the class in question.